src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -58,16 +71,12 @@ export default class RelationshipsPanel extends Vue {
   mounted (): void {
     if (this.draftRelationships.length > 0) {
       this.selectedRelationships.push(...this.draftRelationships)
-    } else {
-      this.selectedRelationships.push() // This is done to validate component immediately (no need to touch)
```

Comment:
There is no need for this now since the new `error` prop will take care of component validation on load right?

Reply:
Yes, that's right @JazzarKarim the component now is always reporting it's validation status.  What we're toggling on and off via the `error` prop is whether those validation errors are shown to the user.

Reply:
This is actually something that I've had trouble dealing with in this and other components. Amazing idea. Thanks Jonathan 😃